lib/appraisal/matrix/rubygems_helper.rb

```diff
@@ -5,20 +5,46 @@
 
 module Appraisal
   module Matrix
-    module RubygemsHelper
-      def versions_to_test(gem_name, minimum_version)
-        # Generate a set to store the versions to test against 
-        versions_to_test = Set.new
-
-        # Load versions from rubygems api
-        URI.parse("https://rubygems.org/api/v1/versions/#{gem_name}.json").open do |raw_version_data|
-          JSON.parse(raw_version_data.read).each do |version_data|
-            version = Gem::Version.new(version_data['number'])
-            versions_to_test << version.segments[0..1].join('.') if version >= minimum_version && !version.prerelease?
+    class RubygemsHelper
+      class << self
+        # Returns a set of versions to test against for a given gem.
+        #
+        # @param gem_name [String, Symbol] The name of the gem.
+        # @param version_requirements [Gem::Requirement] The version requirements for the gem.
+        # @param step [Symbol] The step value.
+        #
+        # @return [Set] A set of versions to test against.
+        def versions_to_test(gem_name, version_requirements, step)
```

Comment:
Nit: `version_requirements` makes me thing this is an array, even though its not. Thoughts on making this singular?
```suggestion
        # @param version_requirement [Gem::Requirement] The version requirements for the gem.
        # @param step [Symbol] The step value.
        #
        # @return [Set] A set of versions to test against.
        def versions_to_test(gem_name, version_requirement, step)
```